environments/icds-cas/inventory/couchdb2.csv

```diff
@@ -1,6 +1,6 @@
-hostname,host_address,group 1,group 2,group 3,var: hostname,L.var: devices,L.var: partitions
-couch0,100.71.184.25,couchdb2,couchdb2_proxy,,MUMGCCWCDPRDCD00,["/dev/vdb"],["/dev/vdb1"]
-couch1,100.71.184.8,couchdb2,,,MUMGCCWCDPRDCD01,["/dev/vdb"],["/dev/vdb1"]
-couch2,100.71.184.9,couchdb2,,,MUMGCCWCDPRDCD02,["/dev/vdb"],["/dev/vdb1"]
-couch3,100.71.184.6,couchdb2,,,MUMGCCWCDPRDCD03,["/dev/vdb"],["/dev/vdb1"]
-couch4,100.71.184.17,couchdb2,,,MUMGCCWCDPRDCD04,["/dev/vdb"],["/dev/vdb1"]
```

Comment:
are these assigned later then?

Reply:
can you share why?
Sorry dont have much context on this.

Reply:
They are only used if the disk needs to be setup by ansible but TCL does the setup. They are a holdover from when we first made the inventory.

Reply:
okay cool. So it was not needed in the first place, but we added it.
thanks